### PR TITLE
RATEST-346: Add a pull request template

### PR DESCRIPTION
**Pull request by dilankavishka**
## Requirements

- [x] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [x] I have tested the changes locally and verified that they work as intended.
- [x] I have added comments to my code where necessary to help others understand my changes.
- [x] I have read and agree to the Contributing Guidelines of this project.

## Summary

Overall, adding a pull request template to the OpenMRS QA framework can help improve the contribution process and ensure that the framework remains a reliable and effective tool for managing healthcare information.

## Screenshots

*None.*

## Related Issue

https://issues.openmrs.org/projects/RATEST/issues/RATEST-346

## Other

*None.*

### CODE REVIEW

**Comment by dilankavishka** [comment]
@Piumal1999 Can you review this please?

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+## Related Issue
+
+[Add the link to the issue you have addressed or created, if applicable]
+
+## Checklist
+
+Please ensure that your pull request meets the following requirements:
+
+[ ] I have tested the changes locally and verified that they work as intended.
```
Comment (mherman22) [inline .github/pull_request_template.md:13]:
```suggestion
- [ ] I have tested the changes locally and verified that they work as intended.
```

Comment (mherman22) [reply .github/pull_request_template.md:13]: add that dashes (`-`) at the beginning of these lines. reference -> https://www.markdownguide.org/extended-syntax/, look for a title called **Task Lists**

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+[ ] I have updated the relevant documentation and/or tests to reflect the changes.
```
Comment (mherman22) [inline .github/pull_request_template.md:15]: do the same here!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+[ ] I have added comments to my code where necessary to help others understand my changes.
```
Comment (mherman22) [inline .github/pull_request_template.md:17]: same here!

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+[ ] I have read and agree to the Contributing Guidelines of this project.
```
Comment (mherman22) [inline .github/pull_request_template.md:19]: same here!

**Review by mherman22 — CHANGES_REQUESTED** [review]

**Comment by Piumal1999** [comment]
I think we can reuse[ this template](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/.github/pull_request_template.md) by just updating the checklist. It also contains clear and concise [instructions as comments](https://github.com/openmrs/openmrs-esm-patient-management/blob/main/.github/pull_request_template.md?plain=1).

And @dilankavishka make sure that you update **this pull request's** description according to the same structure.

File: .github/pull_request_template.md (hunk 4 above)
Comment (dilankavishka) [reply .github/pull_request_template.md:19]: Thank you @mherman22

**Comment by dilankavishka** [comment]
sure! @Piumal1999

**Comment by dilankavishka** [comment]
@Piumal1999  I'm done with the changes. can you please check that?

**Review by Piumal1999 — APPROVED** [review]
Looks good! Thank you @dilankavishka 

cc: @jayasanka-sack

**Review by mherman22 — APPROVED** [review]

**Pull request merged**